src/test/java/de/meggsimum/w3w/What3WordsTest.java

```diff
@@ -104,4 +157,31 @@ public void testWhat3WordsException() {
 		assertTrue(thrown);
 	}
 
+	public void testNonAsciiCharatersFR() {
+		What3Words w3w = new What3Words(API_KEY, "fr");
+		String[] words = {"noël", "étain", "rizière"};
+		double[] coords;
+		try {
+			coords = w3w.wordsToPosition(words);
+			assertEquals(2, coords.length);
+			assertEquals(-21.951124, coords[0]);
+			assertEquals(166.685219, coords[1]);
+		} catch (Exception e) {
+			assertTrue(false);
+		}
+	}
+
+	public void testNonAsciiCharactersDE() {
```

Comment:
Would you mind adding some little API-doc blocks, like the other test methods have?